.github/workflows/setup_microk8s.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This needs to be a separate file as newgrp will not have effect from  setup_environment.sh
+microk8s enable rbac dns storage metallb:10.64.140.43-10.64.140.49
+microk8s kubectl rollout status deployment/hostpath-provisioner -n kube-system
+
+# Juju controller + model
+juju bootstrap --agent-version=2.9.34 --no-gui microk8s uk8sx
+juju add-model kubeflow
```

Comment:
If accepting this, also need to change it in the integration tests where they use the kubeflow model for execution and dumping logs

```suggestion
juju add-model mlflow
```

Reply:
Dont know about this one ... we potentially want to run it with kubeflow in same model.